src/app/common/duck/utils.ts

```diff
@@ -47,6 +47,17 @@ export const handleSelfSignedCertError = (failedUrl: string, dispatch: any) => {
   dispatch(certErrorOccurred(failedUrl));
 };
 
+const testTargetNSName = (value: string) => {
+  if (value?.length < 3 || value?.length > 63) {
+    return 'The namespace name can be between 3 and 63 characters long.';
```

Comment:
```suggestion
    return 'The namespace name should be between 3 and 63 characters long.';
```
@ibolton336 , should this be **should** instead of **can** here?

Reply:
@rayfordj updated!

Reply:
Thanks!